### PR TITLE
Add ORT wrapper for RollingWindowFeaturizer

### PR DESCRIPTION
**Pull request by wangyems**
Update FeaturizerLibrary's commit id
Add ORT wrapper for RollingWindowFeaturizer

### CODE REVIEW

**Comment by wangyems** [comment]
This PR has a pending requirement change

File: onnxruntime/core/graph/featurizers_ops/featurizers_defs.cc [hunk 1]
```diff
@@ -1313,6 +1315,98 @@ void RegisterRobustScalerFeaturizerVer1() {
...
+          {"tensor(int8)", "tensor(uint8)", "tensor(int16)",  "tensor(uint16)", "tensor(int32)", "tensor(uint32)", "tensor(int64)", "tensor(uint64)", "tensor(float)", "tensor(double)"},
+          "No information is available")
+      .TypeConstraint(
+          "OutputT",
+          {"tensor(double)"},
+          "No information is available")
+      .TypeAndShapeInferenceFunction(
+          [](ONNX_NAMESPACE::InferenceContext& ctx) {
+            propagateElemTypeFromDtypeToOutput(ctx, ONNX_NAMESPACE::TensorProto_DataType_DOUBLE, 0);
+            if (hasInputShape(ctx, 1) && hasInputShape(ctx, 2)) {
```
Comment (yuslepukhin) [inline onnxruntime/core/graph/featurizers_ops/featurizers_defs.cc]: >f (hasInputShape(ctx, 1) && hasInputShape(ctx, 2)) { [](start = 13, length = 52)

You should not fail inference if any of the shapes is not available. The output shape depends on the grains_shape but if it is not available do not fail the function. Also if the target shape is not available do not fail. Fail only if it available but the dims are not correct.

File: onnxruntime/featurizers_ops/cpu/rolling_window_transformer.cc [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    const auto* target_tensor(ctx->Input<Tensor>(2));
+    const T* target_data(target_tensor->Data<T>());
+
+    // Prepare the output
+    const auto output_dim_0 = grains_tensor->Shape()[0];
+
+    double* output_data(0);
+    bool has_allocate_output_data = false;
+    std::function<void(OutputType)> callback_fn;
+    callback_fn = [&ctx, &output_data, &has_allocate_output_data, &output_dim_0](OutputType value) -> void {
```
Comment (yuslepukhin) [inline onnxruntime/featurizers_ops/cpu/rolling_window_transformer.cc]: >&output_dim_0 [](start = 66, length = 13)

Should be by value.

Comment (yuslepukhin) [inline onnxruntime/featurizers_ops/cpu/rolling_window_transformer.cc]: >ctx [](start = 20, length = 3)

Should be by value

File: onnxruntime/core/graph/featurizers_ops/featurizers_defs.cc (hunk 1 above)
Comment (wangyems) [reply onnxruntime/core/graph/featurizers_ops/featurizers_defs.cc]: I think I fail it when shape is available but dims are not correct

**Pull request closed without merging**